### DISCUSSION TITLE
How do we motivate growth?

### DISCUSSION THREAD

**Discussion by JackCrumpLeys** [DigitalExtinction/Game#446] [General]
If energy is just a matter of building a bunch of power plants and having some battery drones to keep the army powered what is the point of growing your base and holding territory?

I find that the best battles in these kind of games come from battles over limited resource and then identifying borders and no mans land before the real battle begins. I can see the motivation in expanding the energy grid but this is not super useful and I don't believe is enough motivation.

The way you are shaping it out leads to not that much interesting variation between maps because you just need a well defended power producing base and a bunch of units  with the map layout not having much to do with base layout and defense placement. Without growth I feel that games will get boring and over fast.

**Comment by Indy2222** [comment]
You have some valid points, and this must be well thought through. Obviously, it is near impossible to come up with perfect game dynamics on first try. I think that we will have to experiment with this until we find something which works.

Few technical notes on current ideas:

* Battery drones have limited battery capacity -> motivation to expand electric grid.
* Power grid is not 100% efficient -> motivation to build and defend power plants on multiple places on the map.
* Power transmission is done with lasers (line-of-sight is needed) -> building power hubs at higher altitudes is preferable, terrain introduces inter map variation.
* Some electric generation tech might be location dependent:
  * Wind turbines won't work well in valleys.
  * Nuclear will need source of water for cooling.
  * ?
